README.md

```diff
@@ -34,7 +28,7 @@ Internet browser: Firefox, Google Chrome, Opera...
 Download the code or copy the git repository to your computer.
 
 ### Usage
-Run the index.html file.
+Run the bubble_sort.rb file in your terminal.
```

Comment:
- [ ] Please put ruby keyword before bubble_sort.rb for users to run the algorithm properly.